geometry_msgs/Point point1
geometry_msgs/Point point2
float32 confidence
float32 real_coord
